### DISCUSSION TITLE
Model attribution to Baseline - Low

### DISCUSSION THREAD

**Discussion by shubhambooking2** [google/meridian#188] [General]
Iam currently exploring national model with daily level data. I have been exploring use of knots and google search volume data to account for seasonality and trend, however the baseline seems to perform really bad i.e. the baseline explains not more than 10% of overall kpi. Any suggestions of how to improve the baseline in this context

**Comment by jyklly** [comment]
Exploring different knot values and including query volume is a good approach. If there are any other additional controls you can think of it may be worth exploring including them as well.

It may also be worth examining the priors you are using. Low baseline means high ROIs so if you are using informative ROI priors that could be a cause for low baseline. 

You could also consider setting the ROI prior in terms of total media contribution. See [Set the total media contribution prior](https://developers.google.com/meridian/docs/advanced-modeling/unknown-revenue-kpi#set-total-media-contribution-prio).

For more information about low and negative baseline please see [Baseline is too low or sometimes negative](https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#baseline-low-or-negative).
